test/Conversion/TorchOnnxToTorch/simple_ops_q_to_z.mlir

```diff
@@ -2990,3 +2990,9 @@ func.func @test_scatternd_min(%arg0: !torch.vtensor<[4,4,4],f32>, %arg1: !torch.
   %0 = torch.operator "onnx.ScatterND"(%arg0, %arg1, %arg2) {torch.onnx.reduction = "min"} : (!torch.vtensor<[4,4,4],f32>, !torch.vtensor<[2,1],si64>, !torch.vtensor<[2,4,4],f32>) -> !torch.vtensor<[4,4,4],f32>
     return %0 : !torch.vtensor<[4,4,4],f32>
 }
+
+// CHECK-LABEL: @test_tfidfvectorizer_tf_onlybigrams_levelempty
+func.func @test_tfidfvectorizer_tf_onlybigrams_levelempty(%arg0: !torch.vtensor<[12],si32>) -> !torch.vtensor<[3],f32> attributes {torch.onnx_meta.ir_version = 4 : si64, torch.onnx_meta.opset_version = 9 : si64, torch.onnx_meta.producer_name = "backend-test", torch.onnx_meta.producer_version = ""} {
```

Comment:
have you run this on some test inputs & compared with onnxruntime? Wanna make sure this runs & generates the actual numerical results.

Reply:
SHARK-TestSuite has some op tests you could reference.